packages/make-promise/README.md

```diff
@@ -0,0 +1,3 @@
+# Make Promise
```

Comment:
I think @erights had concerns about the API of my little helper function (specifically that `.res` and `.rej` looked too similar). I didn't spend the time to do a cycle of name-bikeshedding because I didn't think anyone else would use it, but if we're pulling this out to its own module, maybe we should.

Also the README (all the READMEs, really) should have an example of usage.

Reply:
I think we should think about the redesign of the API, but let's do it as a separate PR to not overburden this one.

Reply:
agree. Someone please add an issue so we don't forget this one.

Reply:
That issue has been added: #491